### ISSUE TITLE
Option to disable context menu

### ISSUE THREAD

**Issue by WORMSS** [auchenberg/css-reloader-chrome#30]
All my other extensions have an option to remove the Context Menu, so I have a very nice clean right-click and not bloated with a bunch of stuff I do not require. 

![image](https://user-images.githubusercontent.com/1384537/48892789-c07f1a00-ee36-11e8-9692-4124a3346788.png)

**Comment by xcy7e** [comment]
Yes please. So stupid to take the last item of the context-menu which IT DOES. This is an absolute no-go for webdevs!
